api/orchestrator/orchestrator.proto

```diff
@@ -614,7 +613,7 @@ message AssessmentTool {
 
   // a list of metrics that this tool can assess, referred by their ids
   repeated string available_metrics = 4
-      [ (validate.rules).repeated.items.string.min_len = 1 ];
+      [ (validate.rules).repeated .items.string.min_len = 1 ];
```

Comment:
```suggestion
      [ (validate.rules).repeated.items.string.min_len = 1 ];
```

Reply:
Why did you add the spaces behind .repeated?

Reply:
This is my stupid protobuf formatter, sorry.